### ISSUE TITLE
choice of severity levels?

### ISSUE THREAD

**Issue by 6a68** [seanmonstar/intel#33]
syslog [gives us](http://en.wikipedia.org/wiki/Syslog#Severity_levels) emerg, alert, crit, err, warn, notice, info, debug. These log levels have well-defined and widely understood (by ops type people) meanings. If intel is going to provide a standard set of hierarchical loggers, why not those?

**Comment by seanmonstar** [comment]
They're less understandable to me. I don't know what to do with so many high-severity levels. And I found it's useful to have noiser levels than `debug`. 

@6a68 are you using mozlog?

**Comment by 6a68** [comment]
yup, chronicle is using it. one of the things I'd like to do is map bad stuff to pagerduty, which I can do with the current set of log levels, but I dislike having to tell ops people "I'm going to log using a random subset of the syslog levels, but they mean totally different things, since the highest severity ones are missing"

**Comment by seanmonstar** [comment]
Fair enough. I'd welcome a intel.setLevels() method...
